en-US/firefox/welcome/page6.lang

```diff
@@ -0,0 +1,63 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/welcome/6/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/welcome/6/
+
+
+# HTML page title
+;Make Firefox your default browser
+Make Firefox your default browser
+
+# Main headline
+;Make sure you’re protected, every time you get online
+Make sure you’re protected, every time you get online
+
+
+;When you choose Firefox, you support a better web for you and everyone else. Now take the next step to protect yourself.
+When you choose Firefox, you support a better web for you and everyone else. Now take the next step to protect yourself.
+
+# CTA button
+;Get the Firefox App
+Get the Firefox App
+
+# CTA button
+;Get Firefox on your Phone
+Get Firefox on your Phone
+
+
+;Scan the QR code to get started
+Scan the QR code to get started
+
+# Accessible label for a QR code
+;QR code to scan for Firefox
+QR code to scan for Firefox
+
+
+;Download on the App Store
+Download on the App Store
+
+
+;Choose automatic privacy
+Choose automatic privacy
+
+
+;Companies keep finding new ways to poach your personal data. Firefox is the browser with a mission of finding new ways to protect you.
+Companies keep finding new ways to poach your personal data. Firefox is the browser with a mission of finding new ways to protect you.
+
+
+;Choose freedom on every device
+Choose freedom on every device
+
+
+;Firefox is fast and safe on Windows, iOS, Android, Linux…and across them all. You deserve choices in browsers and devices, instead of decisions made for you.
```

Comment:
Ellipses should have space before and after the punctuation.

Reply:
The ellipses should be attached to the word before, or it could be at the start of a line. I can add a space after though. It looks like it is this way on whatsnew 73 where the copy originally came from, that is without any spacing. I agree it should have a space.

Reply:
You are right, no space after the word, but before the next work, thanks for the fix, 👍